src/store/new-request-module.ts

```diff
@@ -2634,42 +2634,50 @@ export class NewRequestModule extends VuexModule {
 
   @Action
   async getQuickSearch (cleanedName: {exactMatch: string, synonymMatch: string}) {
-    try {
-      this.mutateDisplayedComponent('QuickSearchPending')
-      let encodedAuth = btoa(`${window['quickSearchPublicId']}:${window['quickSearchPublicSecret']}`)
+    const quickSearchPublicId = window['quickSearchPublicId']
+    const quickSearchPublicSecret = window['quickSearchPublicSecret']
```

Comment:
A bummer you had to make these changes in 2 places! Next commit this module will go bye bye.

And by 'bye-bye', i mean placed to the side, because i'm sure we will need to reference it in case of weirdness 😁